.github/workflows/build.yml

```diff
@@ -5,8 +5,22 @@ on:
             - "1.x"
         tags:
             - "*"
+        paths-ignore:
+            - "**.md"
+            - "**.rst"
+            - ".editorconfig"
+            - ".gitignore"
+            - ".npmignore"
+            - "LICENSE"
     pull_request:
         types: [opened, reopened, synchronize]
+        paths-ignore:
```

Comment:
I like the idea of the paths-ignore added here to be more efficient with the ci, but I'm not sure what happens if the checks are required and they don't get run - would the PR be blocked from merging?

@jjangga0214 do you know?

Reply:
I tried this on my fork and the status checks are still required thus the PR cannot be merged in, so I will remove these `paths-ignore` for now, although it is a nice idea.